linux-tools/sqlite/test_files/sample03

```diff
@@ -1,8 +1,8 @@
 seq  name             file                                                      
 ---  ---------------  ----------------------------------------------------------
 0    main                                                                       
-2    e1               /opt/fiv/ltp/testcases/fivextra/sqlite/test_files/examdata
-3    e2               /opt/fiv/ltp/testcases/fivextra/sqlite/test_files/examdata
+2    e1               /usr/local/autotest_base/autotest-client-tests/linux-tools
+3    e2               /usr/local/autotest_base/autotest-client-tests/linux-tools
 ln          fn          exam1       exam2       exam3       exam4     
```

Comment:
since this is the sample output of the testscript which will be using to verify out put of the test script during run time ,the path cannot be changed .it has to be hard coded.